security/maltrail/src/etc/rc.d/opnsense-maltrailserver

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+#
+# $FreeBSD$
+#
+# PROVIDE: opnsense-maltrailserver
+# REQUIRE: SERVERS
+# KEYWORD: shutdown
+#
+
+. /etc/rc.subr
+
+name=maltrailserver
+
+stop_cmd=maltrailserver_stop
+start_cmd=maltrailserver_start
+status_cmd=maltrailserver_status
+rcvar=maltrailserver_enable
+pidfile=/var/run/${name}.pid
+command=/usr/sbin/daemon
+command_args="-f -P /var/run/maltrailserver.pid python2.7 /usr/local/share/maltrail/server.py"
+
+load_rc_config opnsense-maltrailserver
+
+: ${maltrailserver_enable ="NO"}
```

Comment:
```suggestion
: ${maltrailserver_enable="NO"}
```

whoopsie

Reply:
jap, was comparing the both for couple of minutes now .. good eyes :)